src/sfctl/custom_secret_value.py

```diff
@@ -0,0 +1,21 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Commands related to Service Fabric Mesh secret value commands and operations"""
+
+
+def get_secret_value(client, secret_resource_name, secret_value_resource_name, show_value):
+    """structure is meant to make testing easier because testing
```

Comment:
Can you clarify what is referred to by structure?

Reply:
This might not be necessary to say actually, more that on mesh cli it looks like 
    secret_data = client.get(resource_group_name, secret_name, secret_value_resource_name)
    if show_value:
        secret_value = client.list_value(resource_group_name, secret_name, secret_value_resource_name)
        secret_data.value = secret_value['value']
    return secret_data

Which looks simpler but always makes the first request for the general secret info and then an optional request to get the value. But because with how the current generated testing library is set up and only checks against the first request made, by slightly moving it around can check the show value scenario also.